app/src/index.css

```diff
@@ -1,11 +1,23 @@
-
 /* adds Inter & JetBrains Mono */
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&family=JetBrains+Mono:wght@100;200;300;400;500;600;700&display=swap');
+/* @import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&family=JetBrains+Mono:wght@100;200;300;400;500;600;700&display=swap'); */
```

Comment:
We can remove this old font import

Reply:
+1

Reply:
Oops, yeah sorry guys I see I have lots of commented code. Meant to delete all that! Everything is cleaned up.

Reply:
Awesome!